ui/lib/server/page/server_page.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+// import 'package:harmony/spec/spec.dart';
+
+// import '../strings.dart' as strings;
+import '../widget/message.dart';
+
+final messages = [
+  for (var i = 1; i < 11; i++)
+    {
+      'user': 'user $i',
+      'timestamp': DateTime.now().toString(),
+      'message': 'This is a message, wow.'
+    }
+];
+
+class ServerPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+        child: ListView.builder(
+            itemCount: messages.length,
+            itemBuilder: (BuildContext context, int i) {
+              return Container(
```

Comment:
Ideally, the Message UI class would size itself instead of having the Server page control its size. The scrollview should be able to take the incoming size of its children to size itself. Move this Container size to the _Message_ widget.